samples/Riot/sample.riot

```diff
@@ -0,0 +1,27 @@
+<sample>
+  <h3>{ state.message }</h3>
+  <ul>
+    <li each={ tech in state.techs }>
+      { tech.name }
+    </li>
+  </ul>
+
+  <script>
+    export default {
+      state: {
+        message: 'Hello, Riot!',
+        techs: [
+          { name: 'HTML' },
+          { name: 'JavaScript' },
+          { name: 'CSS' }
+        ]
+      }
+    }
+  </script>
+
+  <style>
+    :host { font-size: 2rem }
+    h3 { color: #444 }
+    ul { color: #999 }
+  </style>
+</sample>
```

Comment:
This looks like a standard "Hello world" sample. Can you please replace it with a more realistic real-world sample.

Reply:
Sure no problem I will do it asap. Thank you for your feedback

Reply:
I have updated the template of the PR and added new "real world" examples from the official examples